lib/src/components/NxTransferListHalf/NxTransferListHalf.tsx

```diff
@@ -106,7 +108,7 @@ export default function NxTransferListHalf<T extends string | number = string>(p
         onFilterChange,
         showMoveAll,
         onMoveAll,
-        isSelected,
+        isSelected = true,
```

Comment:
Because we also allow null on optional props, this syntax is unfortunately not usually useful for us - it only catches `undefined`.  You should add unit tests for when `isSelected` is `null`, and you'll see that they don't pass.

Reply:
Thanks for catching it, updated here 15fa580e13a126a1104f8bf8e856d24e4e1ed3df.